### ISSUE TITLE
Crashes on Android 

### ISSUE THREAD

**Issue by LFMAKER** [infobip/mobile-messaging-react-native-plugin#35]
Hello,

We've been receiving some reports from firebase crashlytics of crashes involving this plugin:
Fatal Exception: java.lang.NullPointerException
Attempt to get length of null array
org.infobip.reactlibrary.mobilemessaging.ReactNativeEvent.send

```
Fatal Exception: java.lang.NullPointerException: Attempt to get length of null array
       at org.infobip.reactlibrary.mobilemessaging.ReactNativeEvent.send(ReactNativeEvent.java:28)
       at org.infobip.reactlibrary.mobilemessaging.ReactNativeMobileMessagingModule.initialize(ReactNativeMobileMessagingModule.java:71)
       at com.facebook.react.bridge.ModuleHolder.doInitialize(ModuleHolder.java:236)
       at com.facebook.react.bridge.ModuleHolder.markInitializable(ModuleHolder.java:100)
       at com.facebook.react.bridge.NativeModuleRegistry.notifyJSInstanceInitialized(NativeModuleRegistry.java:129)
       at com.facebook.react.bridge.CatalystInstanceImpl$2.run(CatalystInstanceImpl.java:443)
       at android.os.Handler.handleCallback(Handler.java:914)
       at android.os.Handler.dispatchMessage(Handler.java:100)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:27)
       at android.os.Looper.loop(Looper.java:225)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:226)
       at java.lang.Thread.run(Thread.java:919)
```

Setup:
React Native: 0.66.3
mobile-messaging-react-native-plugin: 4.1.4

Could you assist me in how I can resolve this?

**Comment by johnsoncwb** [comment]
Same problem here.

**Comment by BPatinho** [comment]
We noticed it here since we upgraded `infobip-mobile-messaging-react-native-plugin` from `3.1.1` to `4.1.4`

**Comment by riskpp** [comment]
Thank you for reporting it, we will check.

**Comment by riskpp** [comment]
We fixed it, please check latest version - [4.1.5](https://github.com/infobip/mobile-messaging-react-native-plugin/releases/tag/4.1.5).

**Comment by riskpp** [comment]
Closing, because was fixed
